pyproject.toml

```diff
@@ -24,7 +24,7 @@ classifiers = [
     "Topic :: Home Automation",
     "Topic :: Software Development :: Libraries :: Python Modules",
 ]
-requires-python = ">=3.13.0"
```

Comment:
Please revert this change

Reply:
It's reverted. For some reason it was a dependency for pytest and aiomqtt (win os)